HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/Entity.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicalEditor.Models.MapObjectRelated
+{
+    public class Entity
+    {
+        public long Id { get; set; }
+
+        public Entity()
+        {
+            Id = GenerateId();
+        }
+
+        private long GenerateId()
+        {
+            MainWindow mainWindow = (MainWindow)App.Current.MainWindow;
```

Comment:
The main issue with the IgGenerator was this logic, going back to it is not acceptable. When you run the graphical editor from the other WPF apps you won't be able to access the window you want, because it will no longer be the main window of the app. Try starting the editor from the manager GUI and see what happens.

Reply:
I see the problem. It should be solved by this fix until we establish a connection with DB.

Reply:
I'll resolve conflicts.